mlmc/sample_storage_hdf.py

```diff
@@ -33,36 +33,14 @@ def __init__(self, file_path):
                 for i_level in range(len(self._hdf_object.level_parameters)):
                     self._level_groups.append(self._hdf_object.add_level_group(str(i_level)))
 
-    def _hdf_result_format(self, locations, times):
```

Comment:
Simplified, moved into hdf5.py as it is implementation detail how the format is stored.

Reply:
Moereover the format saving was wrong since it same format has been assumed for all dict elements.